src/components/connection_handler/include/connection_handler/connection_handler_impl.h

```diff
@@ -660,6 +670,16 @@ class ConnectionHandlerImpl
   Connection* GetPrimaryConnection(
       const ConnectionHandle connection_handle) const;
 
+  /**
+   * @brief Handling reason for closed the session.
```

Comment:
```suggestion
   * @brief Handling a reason for closing a session
```

Reply:
@IGapchuk 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/136/commits/957e39c33032718558882f177d2cb17a44624afc